src/addTodoForm.js

```diff
@@ -1,11 +1,19 @@
 import React from "react";
 
-function AddTodoForm() {
+function AddTodoForm(props) {
+
+    function handleAddTodo(event) {
+        event.preventDefault()
+        const todoTitle = event.target.children[1].value
```

Comment:
Nitpick (doesn’t require changes):
You can access the input value via `event.target.title.value`(using the `name` “title” attribute to get the value of the input) and you can reset the form via `event.target.reset()` (using the built-in form [reset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset) method to reset all **uncontrolled** values within the form).